.github/actions/windows-setup-env/action.yml

```diff
@@ -7,6 +7,9 @@ inputs:
   java-version:
     description: "Java version to use in tests"
     default: "8"
+  llvm-version:
+    description: "Java version to use in tests"
```

Comment:
Does this description need changing?

Reply:
Thank you, I've forget to edit copied setting